plugins/keppel/app/javascript/widgets/app/components/utils.jsx

```diff
@@ -63,3 +63,10 @@ export const makeSelectBox = ({ options, value, isEditable, onChange }) => {
 
 const trimEllipsis = (str) =>
   str.substr(-3) === "..." ? str.substr(0, str.length - 3) : str
+
+export const apiStateIsValid = (state) => {
+  if (state == 'deleting') {
+    return false
+  }
+  return true
+}
```

Comment:
This is a strange function name. The state `deleting` is valid.
```suggestion
export const apiStateIsDeleting = (state) => state === 'deleting'
```
Also, it is probably clearer at the callsites what is happening to write `state === SOME_CONSTANT` with a constant for the value `'deleting'` declared here or somewhere else.

Reply:
I was not happy with the naming either and wanted to change it this morning too. The change is in: [2bc61f1](https://github.com/sapcc/elektra/pull/1482/commits/2bc61f198b59ec5c5af4084e266a4efd11073fbf)
I wanted to keep the function more broad at first to include additonal cases in the future, however having one clear state for it is preferable for me too here.